phoenicis-settings/src/main/java/org/phoenicis/settings/Setting.java

```diff
@@ -19,7 +19,7 @@
 package org.phoenicis.settings;
 
 public enum Setting {
-    TERMINAL("tools.linux-terminal");
+    TERMINAL("${tools.linux-terminal:}");
```

Comment:
Are you sure about the `:`?

Reply:
Yes, why not? See <http://www.baeldung.com/spring-value-defaults>

Reply:
Just asking, it looks strange to me